src/kOS/Communication/ProcessorManager.cs

```diff
@@ -53,7 +53,7 @@ public void RunProgramOn(List<Opcode> program, Volume volume)
         {
             kOSProcessor processor = GetProcessor(volume);
             var runCommand = new RunCommand {Program = program};
-            processor.ExecuteInterProcCommand(runCommand);            
+            processor.ExecuteInterProcCommand(runCommand);
```

Comment:
Yeah, I know. For now I've just moved it. I'd prefer to remove it in a separate PR.